### ISSUE TITLE
[Bug] game sometimes freezes randomly

### ISSUE THREAD

**Issue by binnichtaktiv** [Ryujinx/Ryujinx#6949]
### Description of the issue

In this case red dead redemption just freezes sometimes when I play it normally. No mods installed. game version 1.0.4 - same problem with 1.0.1. sometimes the game freezes for 1-2 seconds and then works again but sometimes it freezes forever. only the game.. ryujinx still works and responds. fps drops to 0.00, sometimes i still hear the sound in the game. the cpu and gpu load stays exactly the same also the ram but the game hangs forever. it has gotten worse and worse over time. it used to happen sometimes but now i can't even finish a mission. in the log it says something about "GPU processing thread is too slow, waiting on CPU..." ?? i am playing on iMac m1. Sniper elite 3 hasn't caused any problems so far, neither has mario kart... i tried to disable hypervisor but then rdr crashes when i open it with memory manage mode host selected. only with software can i open it at all but then i only have 8 fps and the delay is crazy (unplayable along with the fps) i don't know what to do. the first time it ran perfectly but now unplayable

### Reproduction steps

Play RDR for a few hours and the crashes will get worse. I changed nothing no mods or changed display settings

### Log file

[Ryujinx_1.1.1334_2024-06-17_00-04-45.log](https://github.com/user-attachments/files/15860023/Ryujinx_1.1.1334_2024-06-17_00-04-45.log)
[Ryujinx_1.1.1334_2024-06-17_00-05-06.log](https://github.com/user-attachments/files/15860024/Ryujinx_1.1.1334_2024-06-17_00-05-06.log)
[Ryujinx_1.1.1334_2024-06-17_00-19-49.log](https://github.com/user-attachments/files/15860025/Ryujinx_1.1.1334_2024-06-17_00-19-49.log)

I think the first two are interesting

### OS

macOS

### Ryujinx version

always up to date (1.1.1334)

### Game version

1.0.1 & 1.0.4 same problem

### CPU

M1

### GPU

Apple GPU 8 cores

### RAM

8GB

### List of applied mods

no mods

### Additional context?

game downloaded from NXBr... idk if im allowed to say that

**Comment by binnichtaktiv** [comment]
if anyone has any tips or knows which settings i can test please let me know

**Comment by MutantAura** [comment]
Duplicate of https://github.com/Ryujinx/Ryujinx/issues/6090

**Comment by binnichtaktiv** [comment]
> Duplicate of #6090

I saw that but like I said I can’t disable Hypervisor or the game will become unplayable
